app/controller/login_control/config.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$client_id = "86opaue7273v2a";
+$client_secret = "wxDOl2fxZXBfjhWM";
```

Comment:
It's a good practice to maintain the configuration or credentials in a different file.
However, add this type of file in `.gitignore` because we don't add confidential details or passwords in repositories.